direnv.el

```diff
@@ -196,6 +196,14 @@ the environment changes."
         (message "direnv: %s (%s)" summary paths)
       (message "direnv: %s" summary))))
 
+(defun direnv--provided-mode-derived-p (mode &rest modes)
```

Comment:
You can also write this as an `if`, and `defalias` your local name to the actual function if it is `fboundp`.

Reply:
thanks, done in 33e3ca084d19045fb4fb034345d8287a6374dc91

had to avoid a conditional `defun` to make `package-lint` happy though (it doesn't infer `defalias` also defines the function).